src/main/java/com/adobe/campaign/tests/integro/phased/PhasedTestListener.java

```diff
@@ -311,164 +318,133 @@ public void onFinish(ITestContext context) {
             PhasedTestManager.deactivateMergedReports();
         }
 
-        if (PhasedTestManager.isMergedReportsActivated()) {
+        boolean isInactive = !PhasedTestManager.isMergedReportsActivated();
+        if (isInactive) {
```

Comment:
Why don't we simply put 

'if (!PhasedTestManager.isMergedReportsActivated())'

instead?

Reply:
I personally use this approach where ever I can because using a variable in between adds to the readability of the code (sometimes the `!` is easy to miss but a well named variable always conveys the intent)